src/applications/vaos/services/appointment/index.js

```diff
@@ -245,37 +245,43 @@ export async function fetchRequestById({ id, useV2 }) {
  * @param {'cc'|'va'} type Type of appointment that is being fetched
  * @returns {Appointment} A transformed appointment with the given id
  */
-export async function fetchBookedAppointment(id, type) {
+export async function fetchBookedAppointment({ id, type, useV2 = false }) {
   try {
     let appointment;
-    if (type === 'va') {
-      appointment = await getConfirmedAppointment(id, type);
-    } else if (type === 'cc') {
-      // We don't have a fetch by id service for cc, so hopefully
-      // the appointment is 13 months in either direction
-      const { data } = await getConfirmedAppointments(
-        type,
-        moment()
-          .add(-395, 'days')
-          .startOf('day')
-          .toISOString(),
-        moment()
-          .add(395, 'days')
-          .startOf('day')
-          .toISOString(),
-      );
-      appointment = data.find(appt => appt.id === id);
-
+    if (!useV2) {
```

Comment:
One way you could make this easier to read is putting the V2 code up first:

```js
let appointment;
if (useV2) {
  appointment = await getAppointment(id);
  return transformVAOSAppointment(appointment);
}

if (type === 'va') {
...
```
Because there's a `return` in the if statement, you can avoid the `else` and make things less nested.

Reply:
Done.